### DISCUSSION TITLE
Convert into contributor

### DISCUSSION THREAD

**Discussion by johnfelipe** [apache/superset#20355] [Q&A / Help]
Pls tell me how can convert to contributor
I want to translate to es, de and it my languages domain strings in Json or other formatting and MD or RST file
And about docker images 
Pls guide me in this matter

**Comment by mdeshmu** [comment]
https://github.com/apache/superset/blob/master/CONTRIBUTING.md
https://preset.io/blog/tutorial-contributing-code-to-apache-superset/

**Comment by johnfelipe** [comment]
I want to contribute with translation
Es
De
It
Fr

But I need help for create PR individual for each lang

I'm using GitHub desktop
